.github/workflows/build_and_test.yml

```
@@ -159,17 +159,13 @@ jobs:
           pip install torch==2.1.0a0+cxx11.abi intel_extension_for_pytorch==2.1.10+xpu -f https://developer.intel.com/ipex-whl-stable-xpu
           cd python/test/unit
           python3 -m pytest -n 8 --verbose --device xpu language/ --ignore=language/test_line_info.py --ignore=language/test_subprocess.py
+          # run runtime tests serially to avoid issues with print buffer handling.
```

Why this test doesn't need to be run serially for NV?
```suggestion
          # run test_subprocess.py serially to avoid issues with print buffer handling.
```

---

I think is due to the way the OpenCL function we call is flushing the output.